rubocop.yml

```diff
@@ -1,3 +1,54 @@
 require:
   - rubocop-performance
   - rubocop-rails
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - "bin/*"
+    - "db/**/*"
```

Comment:
I don't want to ignore db, because rubocop-rails includes rules for migrations

Reply:
@ippachi 
Hmm...
In this project, I'll leave it unset and inherit the default settings.
https://github.com/fusic/rubocop-dobado-omakase/pull/2#discussion_r1917904645